src/style/app.module.css

```diff
@@ -2329,3 +2329,184 @@ button[data-testid='createPostBtn'] {
   width: 1rem;
   margin-right: 8px;
 }
+
+/* PostCard.tsx */
+
+.cardStyles {
+  width: 20rem;
+  background-color: white;
+  padding: 0;
+  border: none !important;
+  outline: none !important;
+}
+
+.cardHeaderPostCard {
+  display: flex;
+  width: 100%;
+  padding-inline: 0;
+  padding-block: 0;
+  flex-direction: row;
+  gap: 0.5rem;
+  align-items: center;
+  background-color: white;
+  border-bottom: 1px solid #dddddd;
```

Comment:
Convert to a variable. There should be some preexisting ones, if not please create it.

Reply:
This isn't fixed